go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/writeas/go-writeas
```

Comment:
I changed the package import path to GitHub because I thought you told me you were moving everything there, but please review in case I'm wrong. Thanks.

Reply:
Yep, this is good :+1: